FreeRTOS_ARP.c

```diff
@@ -85,7 +94,7 @@ _static ARPCacheRow_t xARPCache[ ipconfigARP_CACHE_ENTRIES ];
 
 /** @brief  The time at which the last gratuitous ARP was sent.  Gratuitous ARPs are used
  * to ensure ARP tables are up to date and to detect IP address conflicts. */
-static TickType_t xLastGratuitousARPTime = ( TickType_t ) 0;
+static TickType_t xLastGratuitousARPTime = 0U;
```

Comment:
Why did you remove the cast?

Reply:
That was my suggestion. `TickType_t` is an unsigned type, so it is better to use `0U`. When written as `0U` no further cast is needed, neither for MISRA, nor for a compiler.
I did the same thing when working on MISRA compliance.